packages/office-ui-fabric-react/src/components/Button/examples/Button.Default.Example.tsx

```diff
@@ -1,20 +1,31 @@
 import * as React from 'react';
-import { DefaultButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { DefaultButton, PrimaryButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { Label } from 'office-ui-fabric-react/lib/Label';
 
 export class ButtonDefaultExample extends React.Component<IButtonProps, {}> {
   public render() {
     let { disabled, checked } = this.props;
 
     return (
-      <div className='ms-BasicButtonsExample'>
+      <div className='ms-BasicButtonsExample ms-BasicButtonsTwoUp'>
         <div>
+          <Label>Standard</Label>
           <DefaultButton
             data-automation-id='test'
             disabled={ disabled }
             checked={ checked }
-            iconProps={ { iconName: 'Add' } }
-            description='I am a description'
-            text='Create account'
+            text='Butjon'
+          />
+        </div>
+        <div>
+          <Label>Primary</Label>
+          <DefaultButton
```

Comment:
TBH, if we keep using primary and default button for the next year, that's fine with me. I mostly wanted to get documentation aligned with XD, and needed to support a primary version of compound button. 

If we deprecate both buttons in the future and switch to PushButton, we'll already have the props/styles in place.